definitions/features/candlepin_database.rb

```diff
@@ -0,0 +1,62 @@
+class Features::CandlepinDatabase < ForemanMaintain::Feature
+  CANDLEPIN_DB_CONFIG = '/etc/candlepin/candlepin.conf'.freeze
+
+  include ForemanMaintain::Concerns::BaseDatabase
+
+  metadata do
+    label :candlepin_database
+
+    confine do
+      file_exists?(CANDLEPIN_DB_CONFIG)
+    end
+  end
+
+  def configuration
+    @configuration || load_configuration
+  end
+
+  def validate_available_in_cpdb?
+    check_option_using_cpdb_help('validate')
+  end
+
+  def check_option_using_cpdb_help(option_name, parent_cmd = '')
+    parent_cmd = '/usr/share/candlepin/cpdb' if parent_cmd.empty?
+    help_cmd = "#{parent_cmd} --help |  grep -c '\\-\\-#{option_name} '"
+    execute?(help_cmd)
+  end
+
+  def execute_cpdb_validate_cmd
+    main_cmd = cpdb_validate_cmd
+    unless main_cmd.empty?
+      main_cmd += format_shell_args(
```

Comment:
Missing space here, leads to `no such option: --verbose-u`

Reply:
@iNecas,

Yes, sorry my bad. I have tested code on console only.
Corrected and updated the pull-request.